pkg/ingress/controller/openstack/neutron.go

```diff
@@ -47,6 +47,33 @@ func (os *OpenStack) getFloatingIPs(listOpts floatingips.ListOpts) ([]floatingip
 	return allFIPs, nil
 }
 
+func (os *OpenStack) createFloatingIP(portID string, floatingNetworkID string, description string) (*floatingips.FloatingIP, error) {
```

Comment:
https://github.com/kubernetes/cloud-provider-openstack/blob/35ff40533448368dc1c0de8259aea5754ba5df36/pkg/openstack/loadbalancer.go#L561-L581

should we reuse those functions?

Reply:
I'd vote to not reuse them, just because these are separate controllers with separate codepaths and maintaining them through two LB controllers might be cumbersome.

Reply:
So we duplicate this function and add it in this project ?